FuelLabs/fuel-indexer
# chore: upgrade to beta-4 dependencies

Closes #1167.

### Description

Upgrades the indexer to beta-4-compatible dependencies.

### Testing steps

CI should pass.

As a manual test, run the explorer example and make sure that queries work properly.
